src/ert/_c_wrappers/enkf/enkf_main.py

```diff
@@ -489,8 +471,18 @@ def sample_prior(
                     data = props.values1d.data
                     field_config = config_node.getFieldModelConfig()
                     trans = field_config.get_init_transform_name()
-                    data_transformed = field_config.transform(trans, data)
-
+                    data_transformed = _field_transform(data, trans)
+                    if not storage.field_has_info():
```

Comment:
Create issue explaining how this can be removed if there is an `Experiment`-folder?